### PR TITLE
Pal 618 duplicated simple client examples and clients repo

### PR DESCRIPTION
**Pull request by m78233**
Additional changes needed to this and the Client repo. Will now run the JVM example. Note this pull request will need to be processed at the same time as the PAL 618 pull request in the Client repo.

### CODE REVIEW

File: example-model/pom.xml [hunk 1]
```diff
@@ -52,6 +52,16 @@
             <artifactId>common</artifactId>
             <version>${project.parent.version}</version>
         </dependency>
+        <dependency>
+            <groupId>${project.parent.groupId}</groupId>
+            <artifactId>clients-common</artifactId>
```
[the remark this answers is not in the document]
Comment (m78233) [reply example-model/pom.xml:57]: No, we don't need it.  Over did it on the copy and paste.  Removed it.

**Comment by m78233** [comment]
Re-when to pull it.  This PR has changes for both this repo and the example repo. I would suggest that we wait on this one and get the changes for 639 in first.  Make sure that the changes to the SimpleClient class in that PR are in develop, then check that against the same class in this PR for the client repo.  The point of all would be to make sure that the changes from 639 are migrated over to the client repo.

**Comment by m78233** [comment]
Please note this pull request and the related one for the Palisade-Client repo need to be done at the same time

**Comment by ac74475** [comment]
duplicate PR

**Pull request closed without merging**